### ISSUE TITLE
mimemagic ruby gem yank causing build problems for v2.8.1

### ISSUE THREAD

**Issue by sehqlr** [archivesspace/archivesspace#2257]
## Expected Behavior

When I run `.\build\run bootstrap` in the top-level directory of the project's repo (which is pointed at v2.8.1), I should be able to run the servers afterwards.

## Current Behavior

This is the output for that command:
```
C:\Users\hsam\Projects\archivesspace\build>if bootstrap == bootstrap (java -cp "ant\ant.jar;ant\ant-launcher.jar" org.apache.tools.ant.launch.Launcher bootstrap )  else (
shift 
 java -cp "ant\ant.jar;ant\ant-launcher.jar" org.apache.tools.ant.launch.Launcher bootstrap bootstrap 
) 
Unable to locate tools.jar. Expected to find it in C:\Program Files\Java\jre1.8.0_291\lib\tools.jar
Buildfile: C:\Users\hsam\Projects\archivesspace\build\build.xml

bootstrap-downloads:
      [get] Destination already exists (skipping): C:\Users\hsam\Projects\archivesspace\build\jruby-complete-9.2.12.0.jar
      [get] Destination already exists (skipping): C:\Users\hsam\Projects\archivesspace\build\solr-4.10.4.war
      [get] Destination already exists (skipping): C:\Users\hsam\Projects\archivesspace\build\jetty-runner-9.2.30.v20200428.jar
      [get] Destination already exists (skipping): C:\Users\hsam\Projects\archivesspace\build\jruby-rack-1.1.21.jar
      [get] Destination already exists (skipping): C:\Users\hsam\Projects\archivesspace\solr\lib\icu4j-65.1.jar
      [get] Destination already exists (skipping): C:\Users\hsam\Projects\archivesspace\solr\lib\lucene-analyzers-icu-4.10.4.jar
      [get] Destination already exists (skipping): C:\Users\hsam\Projects\archivesspace\solr\lib\solr-analysis-extras-4.10.4.jar

bootstrap:
   [delete] Deleting directory C:\Users\hsam\Projects\archivesspace\build\gems\gems
   [delete] Deleting directory C:\Users\hsam\Projects\archivesspace\build\gems\bin
   [delete] Deleting directory C:\Users\hsam\Projects\archivesspace\build\gems\specifications
     [java] Successfully installed bundler-2.1.4
     [java] Parsing documentation for bundler-2.1.4
     [java] Done installing documentation for bundler after 8 seconds
     [java] 1 gem installed

set-classpath:

bundler:
     [echo] Fetching gems for ../backend/Gemfile
     [java] Using rake 13.0.1
     [java] Using concurrent-ruby 1.1.7
     [java] Using i18n 0.9.1
     [java] Using minitest 5.14.2
     [java] Using thread_safe 0.3.6 (java)
     [java] Using tzinfo 1.2.7
     [java] Using activesupport 5.2.4.4
     [java] Using ast 2.4.1
     [java] Using atomic 1.0.1 (java)
     [java] Using backports 3.18.2
     [java] Using bcrypt 3.1.7 (java)
     [java] Using builder 3.2.4
     [java] Using bundler 2.1.4
     [java] Using childprocess 3.0.0
     [java] Using choice 0.2.0
     [java] Using coderay 1.1.3
     [java] Using diff-lcs 1.4.4
     [java] Using factory_bot 4.11.1
     [java] Using multipart-post 1.2.0
     [java] Using faraday 1.0.1
     [java] Using faraday_middleware 1.0.0
     [java] Using fastimage 2.2.0
     [java] Using ffi 1.13.1 (java)
     [java] Using jaro_winkler 1.5.4 (java)
     [java] Using jdbc-derby 10.12.1.1
     [java] Using jdbc-mysql 5.1.13
     [java] Using jruby-jars 9.2.12.0
     [java] Using jruby-rack 1.1.21
     [java] Using json 2.3.0 (java)
     [java] Using json-schema 1.0.10
     [java] Using ladle 0.2.0 (java)
     [java] Using method_source 0.9.2
     [java] Using rack 2.2.3
     [java] Using mizuno 0.6.11
     [java] Using multi_json 1.15.0
     [java] Using ruby2_keywords 0.0.2
     [java] Using mustermann 1.1.1
     [java] Using net-http-persistent 2.8
     [java] Using net-ldap 0.16.3
     [java] Using nokogiri 1.10.10 (java)
     [java] Using oai 0.4.0
     [java] Using parallel 1.19.2
     [java] Using parser 2.7.2.0
     [java] Using spoon 0.0.6
     [java] Using pry 0.12.2 (java)
     [java] Using ruby-debug-base 0.11.0 (java)
     [java] Using pry-debugger-jruby 1.2.2 (java)
     [java] Using pry-nav 0.3.0
     [java] Using rack-protection 2.0.5
     [java] Using sequel 5.9.0
     [java] Using rack-session-sequel 0.0.1
     [java] Using rack-test 1.1.0
     [java] Using rainbow 3.0.0
     [java] Using rjack-jackson 1.8.11.0 (java)
     [java] Using tidy 1.1.2
     [java] Using rrtf 1.3.1
     [java] Using rspec-support 3.6.0
     [java] Using rspec-core 3.6.0
     [java] Using rspec-expectations 3.6.0
     [java] Using rspec-mocks 3.6.0
     [java] Using rspec 3.6.0
     [java] Using ruby-progressbar 1.10.1
     [java] Using unicode-display_width 1.6.1
     [java] Using rubocop 0.72.0
     [java] Using rubyzip 1.3.0
     [java] Using rubyXL 3.4.15
     [java] Using rufus-lru 1.0.5
     [java] Using rufus-scheduler 2.0.24
     [java] Using saxerator 0.9.2
     [java] Using saxon-rb 0.8.3 (java)
     [java] Using simplecov-html 0.7.1
     [java] Using simplecov 0.7.1
     [java] Using tilt 2.0.10
     [java] Using sinatra 2.0.5
     [java] Using sinatra-contrib 2.0.5
     [java] Using tzinfo-data 1.2020.1
     [java] Using warbler 2.0.5
     [java] Using zip-zip 0.3
     [java] Bundle complete! 43 Gemfile dependencies, 78 gems now installed.
     [java] Bundled gems are installed into `./gems`

set-classpath:

bundler:
     [echo] Fetching gems for ../frontend/Gemfile
     [java] Fetching gem metadata from https://rubygems.org/.........
     [java] Your bundle is locked to mimemagic (0.3.5), but that version could not be found
     [java] in any of the sources listed in your Gemfile. If you haven't changed sources,
     [java] that means the author of mimemagic (0.3.5) has removed it. You'll need to update
     [java] your bundle to a version other than mimemagic (0.3.5) that hasn't been removed
     [java] in order to install.

BUILD FAILED
C:\Users\hsam\Projects\archivesspace\build\build.xml:207: The following error occurred while executing this line:
C:\Users\hsam\Projects\archivesspace\build\build.xml:408: Java returned: 7

Total time: 1 minute 13 seconds
```

## Possible Solution

I don't know much about Ruby development, but my first thought is to explicitly require a slightly newer version of `mimemagic` then test components to make sure that they don't break with the new version.

## Steps to Reproduce (for bugs)

1. Clone the repository, and change into that directory
2. Switch the git repo to v2.8.1
3. Run `.\build\run bootstrap`

## Context

I've been tasked with writing a new plugin for our hosted instance of archivesspace, and I need to have a local development environment to make that plugin.

## Your Environment

* Version used: 2.8.1
* Operating System and version (desktop or mobile): Windows 10

**Comment by sehqlr** [comment]
I was able to get the build working on my WSL, so this is a Windows specific problem

**Comment by quoideneuf** [comment]
This has been resolved by the Rails team, and as of 3.0.1 the mimemagic gem is no longer a dependency. There is not much that can ben done about the 2.8.1 tag at this point. The 2.8.1 release zip contains the no-longer-available gem but if you try to build or bootstrap using the 2.8.1 tag it will fail to download anything.
